granite.gemspec

```diff
@@ -30,8 +31,8 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'rspec-its', '~> 1.2 '
   s.add_development_dependency 'rspec-rails', '~> 3.6'
   s.add_development_dependency 'rspec_junit_formatter', '~> 0.2'
-  s.add_development_dependency 'rubocop', '~> 0.78.0'
-  s.add_development_dependency 'rubocop-rails', '~> 2.4.1'
-  s.add_development_dependency 'rubocop-rspec', '~> 1.37.0'
+  s.add_development_dependency 'rubocop', '~> 1.0'
```

Comment:
I guess we should lock to minor version and `NewCops: enable`. Right now we wouldn't get any new cops anyway and would get only bugfixes for the current ones, but I doubt there is something to fix.

Reply:
@ojab I set to minor (instead of patch) to encourage us to keep the pace with Rubocop. OK to enable new cops. 👍

Reply:
Then we need `NewCops: enable` (see https://docs.rubocop.org/rubocop/versioning.html#pending-cops), otherwise LGTM.

Reply:
Enabled with this commit: https://github.com/toptal/granite/pull/67/commits/4f0adc5cfe7fdf7d085476d2a159d6e92917adfc